benchmark/kmeans.py

```diff
@@ -230,7 +234,8 @@ def benchmark_data_initialization(
         skdpex_kmeans_engine_module, KMeansEngine=DAAL4PYEngine
     ), sklearnex_config_context(target_offload="cpu"):
         kmeans_timer.timeit(
-            name="daal4py lloyd CPU", engine_provider="sklearn_numba_dpex"
+            name="daal4py lloyd CPU",
+            engine_provider="sklearn_numba_dpex",
```

Comment:
I would revert this change as it is not a semantic one. WDYT?
```suggestion
            name="daal4py lloyd CPU", engine_provider="sklearn_numba_dpex"
```